contracts/IColony.sol

```diff
@@ -446,13 +446,12 @@ contract IColony {
 
   /// @notice Get useful information about specific reward payout
   /// @param _payoutId Id of the reward payout
-  /// @return Reputation root hash at the time of creation
-  /// @return Total colony tokens at the time of creation
-  /// @return Total amount of tokens taken aside for reward payout
-  /// @return Remaining (unclaimed) amount of tokens
-  /// @return Token address
-  /// @return Block number at the time of creation
-  function getRewardPayoutInfo(uint256 _payoutId) public view returns (bytes32, uint256, uint256, uint256, address, uint256);
+  /// @return reputationState Reputation root hash at the time of creation
+  /// @return totalTokens Total colony tokens at the time of creation
+  /// @return amount Total amount of tokens taken aside for reward payout
+  /// @return tokenAddress Token address
+  /// @return blockTimestamp Block number at the time of creation
+  function getRewardPayoutInfo(uint256 _payoutId) public view returns (bytes32 reputationState, uint256 totalTokens, uint256 amount, address tokenAddress, uint256 blockTimestamp);
```

Comment:
I understand you think this is unused but eventually when we come to returning the entire `RewardPayoutCycle` struct, that will return all 5 members regardless and reverting back to it will be a breaking change. Let's keep it as-is

Reply:
Okay, that makes sense; I'll rebase and skip this commit 👍